_episodes/01-numpy.md

```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
 
 In Python, variable names:
 
- - must begin with a letter, and
+ - can include letters, digits, and underscores, but cannot
+   start with a digit
```

Comment:
shall we make an independent bullet point out of`cannot start with a digit`?

Reply:
Sounds good to me.